diff --git a/opl/CMakeLists.txt b/opl/CMakeLists.txt
index 289ca27..90ccd59 100644
--- a/opl/CMakeLists.txt
+++ b/opl/CMakeLists.txt
@@ -21,7 +21,7 @@ if (DEFINED EMSCRIPTEN)
         set_target_properties(opl PROPERTIES LINK_FLAGS "-s USE_SDL=2")
     endif()
 else()
-    target_link_libraries(opl SDL2::SDL2)
+    target_link_libraries(opl SDL2::SDL2-static)
     if(ENABLE_SDL2_MIXER)
         target_link_libraries(opl SDL2_mixer::SDL2_mixer)
     endif()
diff --git a/pcsound/CMakeLists.txt b/pcsound/CMakeLists.txt
index 6f19901..a338986 100644
--- a/pcsound/CMakeLists.txt
+++ b/pcsound/CMakeLists.txt
@@ -17,7 +17,7 @@ if (DEFINED EMSCRIPTEN)
         set_target_properties(pcsound PROPERTIES LINK_FLAGS "-s USE_SDL=2")
     endif()
 else()
-    target_link_libraries(pcsound SDL2::SDL2)
+    target_link_libraries(pcsound SDL2::SDL2-static)
     if(ENABLE_SDL2_MIXER)
         target_link_libraries(pcsound SDL2_mixer::SDL2_mixer)
     endif()
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index aa9d0a4..3db4b0d 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -44,7 +44,7 @@ if (DEFINED EMSCRIPTEN)
         set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -s USE_SDL_NET=2")
     endif()
 else()
-    set(SDL_LINK_LIBS SDL2::SDL2main SDL2::SDL2)
+    set(SDL_LINK_LIBS SDL2::SDL2-static)
     if(ENABLE_SDL2_MIXER)
         list(APPEND SDL_LINK_LIBS SDL2_mixer::SDL2_mixer)
     endif()
@@ -272,7 +272,7 @@ if (DEFINED EMSCRIPTEN)
     set_target_properties(midiread PROPERTIES COMPILE_FLAGS "-s USE_SDL=2")
     set_target_properties(midiread PROPERTIES LINK_FLAGS "-s USE_SDL=2")
 else()
-    target_link_libraries(midiread SDL2::SDL2)
+    target_link_libraries(midiread SDL2::SDL2-static)
 endif()
 
 add_executable(mus2mid mus2mid.c memio.c z_native.c i_system.c m_argv.c m_misc.c d_iwad.c deh_str.c m_config.c)
@@ -282,5 +282,5 @@ if (DEFINED EMSCRIPTEN)
     set_target_properties(mus2mid PROPERTIES COMPILE_FLAGS "-s USE_SDL=2")
     set_target_properties(mus2mid PROPERTIES LINK_FLAGS "-s USE_SDL=2")
 else()
-    target_link_libraries(mus2mid SDL2::SDL2)
+    target_link_libraries(mus2mid SDL2::SDL2-static)
 endif()
diff --git a/src/doom/CMakeLists.txt b/src/doom/CMakeLists.txt
index 6e878a0..95c79a9 100644
--- a/src/doom/CMakeLists.txt
+++ b/src/doom/CMakeLists.txt
@@ -78,7 +78,7 @@ if (DEFINED EMSCRIPTEN)
         set(SDL_FLAGS "${SDL_FLAGS} -s USE_SDL_NET=2")
     endif()
 else()
-    set(SDL_LINK_LIBS SDL2::SDL2)
+    set(SDL_LINK_LIBS SDL2::SDL2-static)
     if(ENABLE_SDL2_MIXER)
         list(APPEND SDL_LINK_LIBS SDL2_mixer::SDL2_mixer)
     endif()
diff --git a/src/heretic/CMakeLists.txt b/src/heretic/CMakeLists.txt
index ac6d2fb..2d25771 100644
--- a/src/heretic/CMakeLists.txt
+++ b/src/heretic/CMakeLists.txt
@@ -64,7 +64,7 @@ if (DEFINED EMSCRIPTEN)
         set(SDL_FLAGS "${SDL_FLAGS} -s USE_SDL_NET=2")
     endif()
 else()
-    set(SDL_LINK_LIBS SDL2::SDL2)
+    set(SDL_LINK_LIBS SDL2::SDL2-static)
     if(ENABLE_SDL2_MIXER)
         list(APPEND SDL_LINK_LIBS SDL2_mixer::SDL2_mixer)
     endif()
diff --git a/src/hexen/CMakeLists.txt b/src/hexen/CMakeLists.txt
index 3eba5d0..8a6f84a 100644
--- a/src/hexen/CMakeLists.txt
+++ b/src/hexen/CMakeLists.txt
@@ -65,7 +65,7 @@ if (DEFINED EMSCRIPTEN)
         set(SDL_FLAGS "${SDL_FLAGS} -s USE_SDL_NET=2")
     endif()
 else()
-    set(SDL_LINK_LIBS SDL2::SDL2)
+    set(SDL_LINK_LIBS SDL2::SDL2-static)
     if(ENABLE_SDL2_MIXER)
         list(APPEND SDL_LINK_LIBS SDL2_mixer::SDL2_mixer)
     endif()
diff --git a/src/setup/CMakeLists.txt b/src/setup/CMakeLists.txt
index c25af65..0876a19 100644
--- a/src/setup/CMakeLists.txt
+++ b/src/setup/CMakeLists.txt
@@ -22,7 +22,7 @@ if (DEFINED EMSCRIPTEN)
         set(SDL_FLAGS "${SDL_FLAGS} -s USE_SDL_MIXER=2")
     endif()
 else()
-    set(SDL_LINK_LIBS, "SDL2::SDL2")
+    set(SDL_LINK_LIBS, "SDL2::SDL2-static")
     if(ENABLE_SDL2_MIXER)
         list(APPEND SDL_LINK_LIBS SDL2_mixer::SDL2_mixer)
     endif()
diff --git a/src/strife/CMakeLists.txt b/src/strife/CMakeLists.txt
index 20d336c..da3d5a5 100644
--- a/src/strife/CMakeLists.txt
+++ b/src/strife/CMakeLists.txt
@@ -79,7 +79,7 @@ if (DEFINED EMSCRIPTEN)
         set(SDL_FLAGS "${SDL_FLAGS} -s USE_SDL_NET=2")
     endif()
 else()
-    set(SDL_LINK_LIBS, "SDL2::SDL2")
+    set(SDL_LINK_LIBS, "SDL2::SDL2-static")
     if(ENABLE_SDL2_MIXER)
         list(APPEND SDL_LINK_LIBS SDL2_mixer::SDL2_mixer)
     endif()
diff --git a/textscreen/CMakeLists.txt b/textscreen/CMakeLists.txt
index 7e33826..fff4a22 100644
--- a/textscreen/CMakeLists.txt
+++ b/textscreen/CMakeLists.txt
@@ -29,5 +29,5 @@ if (DEFINED EMSCRIPTEN)
     set_target_properties(textscreen PROPERTIES COMPILE_FLAGS "-s USE_SDL=2")
     set_target_properties(textscreen PROPERTIES LINK_FLAGS "-s USE_SDL=2")
 else()
-    target_link_libraries(textscreen m SDL2::SDL2)
+    target_link_libraries(textscreen m SDL2::SDL2-static)
 endif()
